.circleci/config.yml

```diff
@@ -118,6 +118,7 @@ commands:
     steps:
       - browser-tools/install-chrome:
           replace-existing: true
+          chrome-version: 93.0.4577.63
```

Comment:
From my understanding of the [dev docs](https://circleci.com/developer/orbs/orb/circleci/browser-tools#commands-install-chrome) for the `browser-tools` orb, it looks like we shouldn't use the `replace-existing` parameter when we specify `chrome-version`. If you agree with this on reading the docs, let's remove the other param.

Reply:
You can click " Show Command Source" on that page to see the actual code that's running. In it, it shows that if Chrome is already installed, it will early exit without this. This check happens before any checks with `chrome-version`.